frontend/src/components/form/api/ApiCheckbox.vue

```diff
@@ -16,7 +16,14 @@ Displays a field as a e-checkbox + write access via API wrapper
       @input="wrapper.on.input"
     >
       <template #append>
-        <api-wrapper-append :wrapper="wrapper" />
+        <ApiWrapperCheckboxAppend
+          v-if="
+            wrapper.hasServerError ||
+            (wrapper.autoSave && wrapper.dirty) ||
```

Comment:
This should be `!wrapper.autoSave`.
The second part `&& wrapper.dirty` depends on my other comment, on whether we want to display a cancel button or not.

Reply:
jest would like that you don't use multi line expressions in the vue html.
please extract this into a computed property

Reply:
you can also leave it, found the problem.
The transformation jest uses does not allow optional chaining (?.), this is the problem in PrintConfigurator.vue and Invitation.vue.